rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```diff
@@ -0,0 +1,42 @@
+[metadata]
+creation_date = "2021/01/09"
+maturity = "development"
+updated_date = "2021/01/09"
+
+[rule]
+author = ["Skoetting"]
+description = "User Added to Privileged Group in Active Directory"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "User Added to Privileged Group in Active Directory"
+risk_score = 21
+rule_id = "5cd8e1f7-0050-4afc-b2df-904e40b2f5ae"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
```

Comment:
```suggestion
type = "query"
```

Reply:
This rule is just as effective in KQL - the benefit of converting is more compatibility across stack versions